test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -1428,3 +1428,89 @@
       initNodes: 1000
       deletingPods: 1000
       measurePods: 1000
+
+# This test case is used to measure queueing hints performance when handling NodeAdd event.
+- name: EventHandlingNodeAdd
+  featureGates:
+    SchedulerQueueingHints: true
+  workloadTemplate:
+  # Collect metrics from all createPods ops.
+  - opcode: startCollectingMetrics
+    name: unschedPods
+    namespaces: [namespace-2, namespace-5, namespace-9, namespace-10, namespace-11, namespace-15]
```

Comment:
This looks tricky to follow. Maybe we could add an optional `namespace` field to `createPods` operations so that we can match those names instead of matching indexes?

Reply:
Good idea! Done